<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Agent</class>
 <widget class="QDialog" name="Agent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>545</height>
   </rect>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:purple;
</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_nom">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>130</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>190</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
        font-size:16px;</string>
   </property>
   <property name="text">
    <string>Nom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>140</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
        font-size:16px;</string>
   </property>
   <property name="text">
    <string>UserName</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cin">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>240</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>300</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
        font-size:16px;</string>
   </property>
   <property name="text">
    <string>Mot de Passe</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>50</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>161</width>
     <height>16</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
        font-size:16px;
</string>
   </property>
   <property name="text">
    <string>Inscrivez Vous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ajouter_Ag">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:30px;
background-color:#262525;
color : grey;</string>
   </property>
   <property name="text">
    <string>Enregistrer </string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../../Sportify-QT-project-master/add.png</normaloff>../../Sportify-QT-project-master/add.png</iconset>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_mdp">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>290</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>250</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
        font-size:16px;</string>
   </property>
   <property name="text">
    <string>CIN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_prenom">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>180</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : transparent;
border:none;
color : #717072;
border-bottom : 1px solid  #717072;
font-size: 18px;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
